### PR TITLE
chore: migrate documentation to new leading main branch

### PR DESCRIPTION
**Pull request by transacid**
Some documentation files still point to the former leading master branch. This commit fixes that.
---
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,22 +46,22 @@ Currently supported project types (use `npx projen new` without a type for a
 list):
 
 <!-- <macro exec="node ./scripts/readme-projects.js"> -->
-* [awscdk-app-ts](https://github.com/projen/projen/blob/master/API.md#projen-awscdktypescriptapp) - AWS CDK app in TypeScript.
```
Comment (eladb) [inline README.md:49]: This needs to be fixed in `scripts/readme-projects.js`

File: API.md [hunk 2]
```diff
@@ -2144,7 +2144,7 @@ addPrebuilds(config: GitpodPrebuilds): void
   * **addComment** (<code>boolean</code>)  Add a "Review in Gitpod" button as a comment to pull requests. __*Default*__: false
   * **addLabel** (<code>boolean</code>)  Add a label once the prebuild is ready to pull requests. __*Default*__: false
   * **branches** (<code>boolean</code>)  Enable for all branches in this repo. __*Default*__: false
-  * **master** (<code>boolean</code>)  Enable for the master/default branch. __*Default*__: true
+  * **main** (<code>boolean</code>)  Enable for the main/default branch. __*Default*__: true
```
Comment (eladb) [inline API.md:2147]: This file is generated from source docs

Comment (eladb) [inline API.md:2147]: This file is generated from source docs

**Review by eladb — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (transacid) [reply README.md:49]: ah ok ;)

Comment (eladb) [reply README.md:49]: Still missing the update to this file

**Review by eladb — CHANGES_REQUESTED** [review]
See note

File: README.md (hunk 1 above)
Comment (transacid) [reply README.md:49]: hmm sorry I think I don't understand what needs to be done there. Isn't the change in `src/inventory.ts` sufficient?

**Review by Chriscbr — APPROVED** [review]

**Pull request merged**